### PR TITLE
Fix npm packages vulnerabilities

### PR DESCRIPTION
**Pull request by idmarjr**
`git clone` and `npm i` gave this:

```
found 48 vulnerabilities (28 low, 7 moderate, 13 high)
```
`npm audit fix` in order to fix vulneravbilities:
```
fixed 2 of 48 vulnerabilities in 14949 scanned packages
```
To fix all 46 remaining vulnerabvilities" `npm audit fix --force`

```
fixed 46 of 46 vulnerabilities in 14949 scanned packages
```

Now `npm audit`return:
``` 
=== npm audit security report ===                                                                                                
found 0 vulnerabilities
in 889635 scanned packages
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,12 +23,24 @@
...
+    "react-scripts": "^3.0.1",
     "styled-components": "^3.4.2",
     "uuid": "^3.1.0"
   },
   "devDependencies": {
     "backstopjs": "^3.8.5"
   },
-  "heroku-run-build-script": true
+  "heroku-run-build-script": true,
+  "browserslist": {
```
Comment (wlsf82) [inline package.json:34]: Do you mind explaining this, please?

File: package.json [hunk 2]
```diff
@@ -23,12 +23,24 @@
...
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
```
Comment (wlsf82) [inline package.json:40]: And this.

**Review by wlsf82 — COMMENTED** [review]
Thanks for your contribution! I left some comments that I'd like you to answer before merging.

File: package.json (hunk 2 above)
Comment (idmarjr) [reply package.json:40]: Everything that I said in [this comment](https://github.com/wlsf82/-walmyr-filho-react/pull/1/files#r289018275) also applies here.

Resuming here:
> When compiling to development, add CSS vendor prefix to last versions of Chrome, Firefox and Safari.

You can read more about this in [browserlist README.md](https://github.com/browserslist/browserslist/blob/master/README.md).

File: package.json (hunk 1 above)
Comment (idmarjr) [reply package.json:34]: Those are [browserlist](https://github.com/browserslist/browserslist) definitions.

`create-react-app` uses `autoprefixer` to parse CSS and add vendor prefixes to rules.
`autoprefixer` [changed from their own way to manage browsers to `browserslist`](https://evilmartians.com/chronicles/autoprefixer-7-browserslist-2-released).

This is the default configs from `browserlist` + `create-react-app`.

Trying to resume:
> When compiling to production add CSS vendor prefix to browsers that have usage >0.2% , are not officially discontinued and are not opera mini (Any version).

You can read more about this in [browserlist README.md](https://github.com/browserslist/browserslist/blob/master/README.md).

**Review by wlsf82 — APPROVED** [review]
Thank you! 🚀

**Comment by wlsf82** [comment]
@idmarjr When trying to rebase and merge I'm getting `This branch cannot be rebased due to conflicts
Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch.`.

Do you mind rebasing your fork on top of the up-to-date upstream repo, so that you can fix any conflict and we can have it merged?

**Comment by idmarjr** [comment]
> @idmarjr When trying to rebase and merge I'm getting `This branch cannot be rebased due to conflicts Rebasing the commits of this branch on top of the base branch cannot be performed automatically due to conflicts encountered while reapplying the individual commits from the head branch.`.
> 
> Do you mind rebasing your fork on top of the up-to-date upstream repo, so that you can fix any conflict and we can have it merged?

@wlsf82 I merged master into `into fix-npm-packages-vulnerabilities`, you should be fine to merge this into your project master now. 😄

**Pull request merged**